opengrok-indexer/src/test/java/org/opengrok/indexer/configuration/SuggesterConfigTest.java

```diff
@@ -0,0 +1,22 @@
+package org.opengrok.indexer.configuration;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SuggesterConfigTest {
+    // TODO add more tests
```

Comment:
Please avoid using `TODO` comments, either remove the `TODO` comment or add the tests.

Reply:
Right. Filed #2611 to track this.